chapters/connectors.tex

```diff
@@ -107,9 +107,13 @@ \subsection{Expandable Connectors}\label{expandable-connectors}
 This applies to both variables of simple types, and variables of structured types.
 
 Then connections containing expandable connectors are elaborated:
-\begin{itemize}
+\begin{enumerate}
+\item
+  If a connect-equation references a potentially present component (either as the complete argument or as part of the argument) it will be marked as being present and will be treated below as declared.
```

Comment:
```suggestion
  If a connect-equation references a potentially present component \emph{as part of the argument} it will be marked as being present, which will allowing a connection to an undeclared connector inside it.
  The rule does not apply for the complete argument.
```
As discussed.